src/Calculator/Views/DateCalculator.xaml

```diff
@@ -593,7 +595,8 @@
             <!-- Resulting Date -->
             <TextBlock x:Uid="DateLabel"
                        Grid.Row="8"
-                       Foreground="{ThemeResource SystemControlPageTextBaseMediumBrush}"/>
+                       Foreground="{ThemeResource SystemControlPageTextBaseMediumBrush}"
+                       IsTextSelectionEnabled="True"/>
```

Comment:
Was this a mistake?  None of the other labels have text selection enabled so this feels out-of-place.

Reply:
This one is not a standard label, it displays the result of the feature "Add or substract days"

Reply:
I'm not sure I understand why that makes this label special?
In this first image, the red shows selectable text and the blue shows non-selectable labels:
![image](https://user-images.githubusercontent.com/23107231/54409029-da709100-46a1-11e9-9444-72f48e14a995.png)

Compare with this second image:
![image](https://user-images.githubusercontent.com/23107231/54409081-1572c480-46a2-11e9-9d31-809b54718bca.png)

I don't understand why the 'Date' label is selectable text.  In the first image, 'Difference' was not selectable although it was also displaying the result of the feature.  At least we can say that we're inconsistent between each feature.  Either the label for the result should be selectable in both or not selectable in both.

Second to that, I'm arguing that the 'Date' label should not be selectable.  The *only* action the user can take by selecting the label is to copy the label alone, meaning they can't grab `Date: Wednesday, May 19, 2021` as a single run.  What is special about the label then?  What is helped by allowing it to be selectable where it wasn't before?  Why does this label deserve to be selectable but other labels like 'From' and 'To' are not selectable?

Reply:
> I'm arguing that the 'Date' label should not be selectable. 

I agree with @danbelcher-MSFT. Field labels should not be selectable, only the values/output.

Reply:
I was confused with `DateDiffAllUnitsResultLabel`, I agree, this label should not be selectable!

Reply:
😅